blockly-core/core/ui/contextmenu.js

```diff
@@ -136,16 +133,6 @@ Blockly.ContextMenu.optionToDom_ = function(text) {
 };
 
 /**
- * Hide the context menu.
- */
-Blockly.ContextMenu.hide = function() {
```

Comment:
hah, yeah, I'm not sure why this was added but it definitely shouldn't have been.

Unfortunately, none of this stuff has been touched this the current repo was created.